skbio/stats/distance/_base.py

```diff
@@ -853,6 +872,38 @@ def __getitem__(self, index):
         else:
             return self.data.__getitem__(index)
 
+    def _validate_ids(self, data, ids):
```

Comment:
Would it be possible to add in unit tests for the various paths through this function?

Reply:
OK.
Note, however, that the code is already indirectly tested as part of the _validate unit test.

Reply:
Added.